Library/Homebrew/completions/bash.erb

```diff
@@ -0,0 +1,162 @@
+<%
+# To make changes to the completions:
+#
+# - For changes to a command under `COMMANDS` or `DEVELOPER COMMANDS` sections):
+#   - Find the source file in `Library/Homebrew/[dev-]cmd/<command>.{rb,sh}`.
+#   - For `.rb` files, edit the `<command>_args` method.
+#   - For `.sh` files, edit the top comment, being sure to use the line prefix
+#     `#:` for the comments to be recognized as documentation. If in doubt,
+#     compare with already documented commands.
+# - For other changes: Edit this file.
+#
+# When done, regenerate the completions by running `brew man`.
+%>
```

Comment:
Nice, maybe could even add this to a doc somewhere (if it hasn't already)

Reply:
You know what, this information can be found (somewhat) in the [external commands documentation](https://docs.brew.sh/External-Commands). I also have a separate PR open to improve this. I'd say if there's something you think is missing, leave a comment/review over in https://github.com/Homebrew/brew/pull/10333 and I'll try to improve it there.